index.html

```diff
@@ -0,0 +1,209 @@
+          <select>
+            <option value="six-month">6 months</option>
+            <option value="one-year">One year</option>
+            <option value="two-years">Two Years</option>
+            <option value="three-years">Three Years</option>
+            <option value="four-years">Four Years</option>
+            <option value="five-years">Five Years</option>
+ 
+          </select>
+        </div>
+
+     </div>
+
+     <div class="row-tab">
+      <div class="label-select">
+        <label>How many SMS do you send per day?</label>
+      </div>
+      <div class="Select-input">
+        <select>
+          <option value="often">0-10</option>
+          <option value="10-20">10-20</option>
+          <option value="20*30">20-30</option>
+          <option value="30-40">30-40</option>
+          <option value="40-50">40-50</option>
+          <option value="50-55">50-55</option>
+
+        </select>
+      </div>
+
+   </div>
+
+   <div class="row-tab">
+    <div class="label-select">
+      <label>How much money do You spend on mobile services per month?</label>
+    </div>
+    <div class="Select-input">
+      <select>
+        <option value="$50">$50-$100</option>
+        <option value="$100">$100-$150</option>
+        <option value="$150">$150-$200</option>
+        <option value="$250">$250-$300</option>
+        <option value="$300">$300-$350</option>
+        <option value="$350">$350-$400</option>
+
+      </select>
+    </div>
+
+ </div>
+ <div class="row-tab">
+  <div class="label-select">
+    <label>You often use your phone to:</label>
+  </div>
+  <div class="Select-input">
+    <select>
+      <option value="six-month">SMS</option>
+      <option value="one-year">Calls</option>
+      <option value="two-years">Social Media</option>
+      <option value="three-years">WAP(Mobile internet)</option>
+      <option value="four-years">Online Lesson</option>
+      <option value="five-years">Others</option>
+
+    </select>
+  </div>
+
+</div>
+
+<div class="row-tab">
+  <div class="label-select">
+    <label> Do You use your mobile phone while driving?</label>
+  </div>
+  <div class="Select-input">
+    <select>
+      <option value="never">Never</option>
+      <option value="for-sms">Just when sms comes</option>
+      <option value="forscalls">Only when answering a call</option>      
+
+    </select>
+  </div>
+</div>
+
+  <div class="form-survey">
+    <label for="commets">Any comments or addional information?
+
+    </label>
+    <textarea name="comments" id="comments" placeholder="Enter your commen here......"></textarea>
+  </div>
+
+        
+  </div>
+  <button type="submit">Submit</button>
+
+</form>
+
+    
+
+  <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
- [ ] [OPTIONAL] I think it will be nice if you asked the student to remove useless comments because they make the code base ugly